layersvt/device_simulation.cpp

```diff
@@ -48,6 +48,8 @@
 #include <fstream>
 #include <mutex>
 
+using namespace std;
```

Comment:
I believe this is against convention.  Go ahead and include std:: in your uses of the library.